app/lib/events/importers/alert_importer.rb

```diff
@@ -89,7 +83,7 @@ def send_email(alert)
       end
 
       def publish_events(alert)
-        Alerts::PublishAlertEventService.run!(alert) if alert.persisted?
```

Comment:
Why is the condition removed?

Reply:
We have two kinds of alerts: emails and web alerts. Web alerts are stored in DB, because they need to be shown in a screen; but email alerts just generate the notification and don't store anything in DB.

This condition prevented email alert notifications to be sent from the new system because the alert is not in DB in this scenario. So the old code is sending web alert notifications using the new system and email alert notifications using the old system.

In order to be able to use the new system for all alerts, I had to remove this condition and modify the event to ensure it includes all info the worker needs to send the notification.